packages/cadl-ranch-specs/http/type/model/visibility/main.tsp

```diff
@@ -48,7 +48,10 @@ Expected response body:
 ```
 """)
 @get
-op getModel(@body input: VisibilityModel): VisibilityModel;
+op getModel(@body input: VisibilityModel): {
+  @body
```

Comment:
what is the change you're seeing without this change?

Reply:
for tsp 0.56.0, it will do visibility filtering for body if not explicitly `@body`.

Reply:
> what is the change you're seeing without this change?

There is behavior change in compiler (or http/rest) lib for the response model. 0.55 would give `VisibilityModel`, but 0.56 would give an anonymous model with non-"read" properties filtered out.
https://github.com/microsoft/typespec/issues/3225